sqlmesh/dbt/source.py

```diff
@@ -86,12 +87,22 @@ def canonical_name(self, context: DbtContext) -> str:
 
     @property
     def relation_info(self) -> AttributeDict:
+        external_location = (
+            self.source_meta.get("external_location", None) if self.source_meta else None
+        )
+        external = (
+            {"external": external_location.replace("{name}", self.table_name)}
+            if external_location
+            else {}
+        )
```

Comment:
I suggest we call this extras, so the dict can cover other cases. I'm adding additional databricks support, which is similar to this, but different fields from a different dbt config dict and imo it makes sense to use the same dict var for both.

Reply:
yes changed this to `extras` this does make more sense